src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public abstract class Figure implements Area,Drawable {
+    private final String color;
+
+    public Figure(String color) {
+        this.color = color;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println(", area: " + area() + ", color: " + color);
+    }
+
+    @Override
+    public double area() {
```

Comment:
The `area()` method in the `Figure` class should not have an implementation (returning 0), as each specific figure will have its own way to calculate the area. This method should be abstract to enforce implementation in the subclasses.